.changeset/modern-taxis-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@penumbra-zone/types': minor
+---
```

Comment:
question: should we take those packages you've listed, create a new package, make those specific files private, and only expose pnum?

Reply:
related: if we make the other stuff private, we'll need to find the previous usages and convert them to pnum